app/src/marketplace/containers/EditProductPage/SharedSecrets.jsx

```diff
@@ -0,0 +1,28 @@
+// @flow
+
+import React from 'react'
+import cx from 'classnames'
+import ScrollableAnchor from 'react-scrollable-anchor'
+import { Translate } from 'react-redux-i18n'
+
+import SharedSecretEditor from './SharedSecretEditor'
+
+import styles from './productStreams.pcss'
+
+type Props = {
+    className?: string,
+}
+
+const SharedSecrets = ({ className }: Props) => (
+    <ScrollableAnchor id="shared-secrets">
```

Comment:
And this should be just `<section id="shared-secrets">`, `ScrollableAnchor ` is not needed since it's using `'react-scrollspy'` and it picks them by the ids.